indexing-service/src/test/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTaskTest.java

```diff
@@ -569,10 +735,10 @@ public void testKillMultipleUnusedSegmentsWithDifferentMaxUsedStatusLastUpdatedT
 
     final List<DataSegment> unusedSegments =
         getMetadataStorageCoordinator().retrieveUnusedSegmentsForInterval(
-                DATA_SOURCE,
-                umbrellaInterval,
-                null,
-                null
+            DATA_SOURCE,
+            umbrellaInterval,
+            null,
+            null
             );
```

Comment:
Indentation seems off. Closing brace should have smaller indentation than the preceding args.